.pre-commit-config.yaml

```diff
@@ -1,6 +1,9 @@
 ---
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+#
+# Run the following command to set up the pre-commit git hook scripts:
+# $ pre-commit install
```

Comment:
Ultimately, we should start writing a contributor/developper's guide. But this is good for now.